### PR TITLE
Fixing Issue #4

### PR DESCRIPTION
**Pull request by codingpixi**
I believe I have done everything to close issue #4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,35 +1,53 @@
...
-      </ul>
+  <header>
+
+    <img class="logo front logoDisplay" src="img/logo.png" alt="Black Orlando Tech logo with a robot head in place of the letter O">
+
+  </header>
+
+  <div class="container">
+    <h1>Why Web Development?</h1>
+    <!--sample title heading-->
```
Comment (nellarro) [inline index.html]: 🙃 Not sure if this comment here is necessary?

File: main.css [hunk 2]
```diff
@@ -78,31 +90,67 @@ p {
...
 @media only screen and (min-width: 401px) and (max-width: 960px) {
   body {
-    background-color: #60B2E5; /* Blue*/
+    background-color: #60B2E5;
+    /* Blue*/
+  }
+  /* Logo should flip on the y axis */
+  .logo {
+    transform: rotateY(180deg);
+    transition-duration: 1s;
```
Comment (nellarro) [inline main.css]: 🙃 This should probably match up with the duration of the other animations.

**Review by nellarro — COMMENTED** [review]
Great job! Leaving this up for a bit in case you want to make any changes and I can merge it in before class.

File: index.html (hunk 1 above)
Comment (codingpixi) [reply index.html]: Ok. I will remove that now. Thx 😊

File: main.css (hunk 2 above)
Comment (codingpixi) [reply main.css]: Yep, didn't even consider that. Thx and changing that now too

**Comment by codingpixi** [comment]
I've made those updates and pushed the changes. From what I read this will update the PR. Please let me know if I need to do anything else 😊

**Pull request merged**